docs/example.lua

```diff
@@ -10,7 +10,15 @@ local s = smartfs.create("smartfs:form", function(state)
 	usr:setBackground("halo.png")
 	state:field(7.25,1.25,3,1,"txt","Textbox")
 	state:image(0,0,2,2,"img","default_stone.png")
-	state:toggle(0,2,3,1,"tg",{"plenty..","of..","custom..","elements"})
+	local toggle = state:toggle(0,2,3,1,"tg",{"plenty..","of..","custom..","elements"})
+	toggle:onToggle(function(self, state, player)
+		if state:get("ta"):getVisible() == false then
```

Comment:
The sample works as expected. If hidden then show up, else hide. The first "getVisible" is the check if the "ta" field is hidden.
About usage of local references: It is a sample how the smartfs could be used. If I need the reference more then 1-2 times you are right, usage of local reference is better. But using of local references are already shown multiple times, like local usr, local toggle... The new sample is a showcase for the get() method too ;)